### ISSUE TITLE
Can't build App, Error: import module hook setuptools.extern.six.moves, No module named _vendor.six

### ISSUE THREAD

**Issue by felix-r** [pyinstaller/pyinstaller#3777]
After I updated to version 3.4 I get the following error when bundling the application: 

```
17692 INFO: Processing pre-safe import module hook   setuptools.extern.six.moves
Traceback (most recent call last):
  File "<string>", line 2, in <module>
ImportError: No module named _vendor.six
Traceback (most recent call last):
  File "c:\users\s8570974\appdata\local\continuum\anaconda2\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
```
...
```
  File "c:\users\s8570974\appdata\local\continuum\anaconda2\lib\site-packages\PyInstaller\depend\analysis.py", line 260, in _safe_import_module
    hook_module.pre_safe_import_module(hook_api)
  File "c:\users\s8570974\appdata\local\continuum\anaconda2\lib\site-packages\PyInstaller\hooks\pre_safe_import_module\hook-setuptools.extern.six.moves.py", line 34, in pre_safe_import_module
    for real_module_name, six_module_name in real_to_six_module_name.items():
AttributeError: 'str' object has no attribute 'items'
```
My dev env is conda. For the build I used:
PyInstaller: 3.5.dev0+1532d3668
Python: 2.7.14
Platform: Windows-10-10.0.14393

With version 3.3.1 the application builds with no errors and runs smoothly.

**Comment by sheebz** [comment]
I have same problem.  Was able to work around by running:
 `pip install --force-reinstall --no-binary :all: pyinstaller` 
from [this issue](https://github.com/pyinstaller/pyinstaller/issues/3777)

**Comment by davcastroruiz** [comment]
@sheebz ! thank you :) i was trying a lot of things.

**Comment by rllola** [comment]
Same for me but on travis... 
https://travis-ci.org/rllola/ZeronetBrowser/jobs/443361238

**Comment by htgoebel** [comment]
Duplicate of #3806. Should be fixed now in 0ffa574bf87e2e37b0df2a4e2d60cb616f9ff2f2

**Comment by alex-s-k-v** [comment]
tried this way but it didn't help in PyCharm
following steps were successfull:
1) install module six
2) force reintall module setuptools
3) install module pefile
4) install or force reinstall pyinstaller

**Comment by elibroftw** [comment]
> tried this way but it didn't help in PyCharm following steps were successfull:
> 
>     1. install module six
> 
>     2. force reintall module setuptools
> 
>     3. install module pefile
> 
>     4. install or force reinstall pyinstaller

Thank you so much. Fixed my Python 3.10 build issues.
